_maps/shuttles/shiptest/independent_schmiedeberg.dmm

```diff
@@ -2379,7 +2381,7 @@
 /obj/item/assembly/flash/handheld,
 /obj/item/door_remote/chief_medical_officer,
 /obj/item/pet_carrier,
-/obj/item/wallframe/defib_mount,
```

Comment:
Is there a defib on the ship that this was intended for or was it truly random

Reply:
yeah, there's not a single defib on this ship. but there's a defib mount in the locker. odd!